server/bin/pbench-backup-tarballs.py

```diff
@@ -0,0 +1,448 @@
+        controller = os.path.basename(controller_path)
+
+        # This function call will handle all the local backup related
+        # operations and count the number of success and failure.
+        local_backup_result = backup_to_local(
+            lb_obj, logger, controller_path, controller, tb, tar, resultname, archive_md5, archive_md5_hex_value)
+
+        if local_backup_result == Status.SUCCESS:
+            nbackup_success  += 1
+        elif local_backup_result == Status.FAIL:
+            nbackup_fail += 1
+        else:
+            assert False, "Impossible situation, local_backup_result = {}".format(
+                local_backup_result)
+
+        # This function call will handle all the S3 bucket related
+        # operations and count the number of success and failure.
+        s3_backup_result = backup_to_s3(
+            s3_obj, logger, controller_path, controller, tb, tar, resultname, archive_md5_hex_value)
+
+        if s3_backup_result == Status.SUCCESS:
+            ns3_success += 1
+        elif s3_backup_result == Status.FAIL:
+            ns3_fail += 1
+        else:
+            assert False, "Impossible situation, s3_backup_result = {}".format(
+                s3_backup_result)
+
+        if local_backup_result == Status.SUCCESS \
+                and s3_backup_result == Status.SUCCESS:
+            # Move tar ball symlink to its final resting place
+            _rename_tb_link(tb, os.path.join(
+                controller_path, _linkdest), logger)
+
+    return Results(ntotal=ntotal,
+                   nbackup_success=nbackup_success,
+                   nbackup_fail=nbackup_fail,
+                   ns3_success=ns3_success,
+                   ns3_fail=ns3_fail,
+                   nquaran=nquaran)
+
+
+def main():
+    cfg_name = os.environ.get("CONFIG")
+
+    if not cfg_name:
+        print("{}: ERROR: No config file specified; set CONFIG env variable or"
+              " use --config <file> on the command line".format(_NAME_),
+              file=sys.stderr)
+        return 2
+
+    try:
+        config = PbenchConfig(cfg_name)
+    except BadConfig as e:
+        print("{}: {}".format(_NAME_, e), file=sys.stderr)
+        return 1
+
+    logger = get_pbench_logger(_NAME_, config)
+
+    # Add a BACKUP and QDIR field to the config object
+    config.BACKUP = config.conf.get("pbench-server", "pbench-backup-dir")
+    config.QDIR = config.get('pbench-server', 'pbench-quarantine-dir')
+
+    # call the LocalBackupObject class
+    lb_obj = LocalBackupObject(config)
+
+    # call the S3Config class
+    s3_obj = S3Config(config)
+
+    lb_obj, s3_obj = sanity_check(lb_obj, s3_obj, config, logger)
+
+    if lb_obj is None and s3_obj is None:
+        return 3
+
+    logger.info('start-{}'.format(config.timestamp()))
+
+    # Initiate the backup
+    counts = backup_data(lb_obj, s3_obj, config, logger)
+
+    result_string = ("Total processed: {}, "
+                     "Locally backed-ed succesfully: {}, "
```

Comment:
"backed-up"
Or maybe (see below): "Local backup successes: " and "S3 upload successes: "